### PR TITLE
Implement Desktop Version 

### PR DESCRIPTION
**Pull request by shahzaman900**
### Implement Desktop Version 
**In this PR I implement**
1. The header section
2. The headline section
3. Works section 
4. About my self section
5. contact form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,54 +16,46 @@
...
+								alt="Linkedin"
+							/></a></li>
+							<li><a href=""><img class="icons" src="./images/madiam.svg" alt="madiam" /></a></li>
+							<li><a href=""><img class="icons" src="./images/github.svg" alt="github" /></a></li>
+							<li><a href=""><img class="icons" src="./images/victor.svg" alt="victor" /></a></li>
+						</ul>
 				</div>
 			</div>
 		</section>
 		<section id="works">
```
Comment (souad988) [inline index.html:56]: Great work done in this milestone congrats, I have only on thing to mention in this section the image is well positioned 👍🏼  but the info section is not correctly aligned , the margins between items is less than the design and also it doens't cover the same space refer to the images below

image (10)
image (11)

File: index.html [hunk 2]
```diff
@@ -195,35 +187,29 @@ <h2>Multi-Post
...
-							class="icons"
-							src="./images/Linkedin icon.svg"
-							alt="Linkedin"
-						/></a></li>
-						<li><a href=""><img class="icons" src="./images/madiam.svg" alt="madiam" /></a></li>
-						<li><a href=""><img class="icons" src="./images/github.svg" alt="github" /></a></li>
-						<li><a href=""><img class="icons" src="./images/victor.svg" alt="victor" /></a></li>
-					</ul>
-				</div>
+				<h1>About Myself</h1>
```
Comment (souad988) [inline index.html:192]: In this section languages list need's to change display direction to horizontal on desktop kindly fix it 

image (12)
image (13)

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

242722065-db428b0c-8dfc-49c4-a156-42f227e32868



## Highlights
Good use of grid and flexbosx to layout components
Good PR description

## Required Changes ♻️
_Check the comments under the review._
Also please don't forget to align the contact form button the same way as in the design I'm adding this comment here since i don't have access to it in the file to add inline comment , please refer to the images below

image (14)
image (15)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bantealem — COMMENTED** [review]
Hi @sheezu900 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a Linters error. Kindly resolve is and request a review for the second time.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by roshan-bajgain — APPROVED** [review]
Hi @sheezu900, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Good job on implementing the changes ✔️ 
## Optional suggestions
- [x] Everyone is facing the `webhint` problem. Hope it will be resolve as soon as possible.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**